packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -242,13 +243,23 @@ abstract class CachingAssetBundle extends AssetBundle {
       return _standardMessageData[key];
     }
 
-    return compute((_) async {
-      final ByteData data = await load(key);
-      final SynchronousFuture<dynamic> result = 
-        SynchronousFuture<dynamic>(const StandardMessageCodec().decodeMessage(data));
-      _standardMessageData[key] = result;
-      return result;
-    }, null);
+    // ~10,000 ms
+    // final ByteData data = await load(key);
```

Comment:
Ahh yeah, we'd have to copy the data back to the compute isolate to do the decode since ByteData is mutable. For now its probably best to keep it on the main thread.